configs/opencv-ovms/scripts/grpc_postprocess.py

```diff
@@ -61,4 +61,10 @@ def postProcessBit(response, duration):
     output = as_numpy(response, "output_1")
     nu = np.array(output)
     print('Processing time: {:.2f} ms; fps: {:.2f}'.format(round(np.average(duration), 2),round(1000  / np.average(duration), 2)))
+    return output
+
+def postProcessYolov5s(response, duration):
+    output = as_numpy(response, "326/sink_port_0")
+    nu = np.array(output)
+    print('Processing time: {:.2f} ms; fps: {:.2f}'.format(round(np.average(duration), 2),round(1000  / np.average(duration), 2)))
```

Comment:
just curious, should it be bnp.average(duration)/1000 since the duration is in ms and we want it to be in second @brian-intel @jim-wang-intel